src/test/java/edu/ksu/canvas/attendance/services/CanvasApiWrapperServiceUTest.java

```diff
@@ -63,7 +63,7 @@ public void setup() throws NoLtiSessionException {
     @SuppressWarnings("unchecked")
     public void getEnrollmentsFromCanvas_HappyPath() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)